src/Raven.Client/Documents/Operations/DatabaseStatistics.cs

```diff
@@ -70,6 +71,8 @@ public class DatabaseStatistics
         public string Pager { get; set; }
 
         public DateTime? LastIndexingTime { get; set; }
+
+        public ChangeVectorEntry[] LastChangeVector { get; set; }
```

Comment:
`DatabaseChangeVector` might be better. It isn't actually the _last_ change vector. it is just the highest we have seen / extrapolated.